src/storage-preview/azext_storage_preview/_validators.py

```diff
@@ -353,6 +357,31 @@ def validator(namespace):
     return validator
 
 
+def get_file_path_validator_v2(default_file_param=None):
```

Comment:
same as above. Could you just replace previous get_file_path_validator?

Reply:
OK